systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -744,6 +749,130 @@ void testNetworkPoliciesWithTlsListener() throws Exception {
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 3, 1).done();
+        LOGGER.info("Getting IP of the bootstrap service");
+
+        String ipOfBootstrapService = kubeClient().getService(KafkaResources.bootstrapServiceName(CLUSTER_NAME)).getSpec().getClusterIP();
+
+        LOGGER.info("Kafka connect without config {} will not connect to {}:9093", "ssl.endpoint.identification.algorithm", ipOfBootstrapService);
+
+        KafkaConnect kafkaConnect = testMethodResources().kafkaConnectWithoutWait(testMethodResources().defaultKafkaConnect(CLUSTER_NAME, 1)
+                .editMetadata()
+                    .addToLabels("type", "kafka-connect")
+                .endMetadata()
+                .editSpec()
+                    .withNewTls()
+                        .addNewTrustedCertificate()
+                            .withSecretName(CLUSTER_NAME + "-cluster-ca-cert")
+                            .withCertificate("ca.crt")
+                        .endTrustedCertificate()
+                    .endTls()
+                    .withBootstrapServers(ipOfBootstrapService + ":9093")
+                .endSpec()
+                .build());
+
+        StUtils.waitUntilPodIsPresent(CLUSTER_NAME + "-connect");
+
+        String kafkaConnectPodName = kubeClient().listPods("type", "kafka-connect").get(0).getMetadata().getName();
+
+        StUtils.waitUntilPodIsInCrashLoopBackOff(kafkaConnectPodName);
+
+        assertThat("CrashLoopBackOff" ,is(kubeClient().getPod(kafkaConnectPodName).getStatus().getContainerStatuses().get(0)
+                .getState().getWaiting().getReason()));
+
+        replaceKafkaConnectResource(CLUSTER_NAME, kc -> {
+            kc.getSpec().getConfig().put("ssl.endpoint.identification.algorithm", "");
+        });
+
+        LOGGER.info("Kafka connect with config {} will connect to {}:9093", "ssl.endpoint.identification.algorithm", ipOfBootstrapService);
+
+        TestUtils.waitFor("Waiting till kafka connect status will be 'Ready'", Constants.GLOBAL_POLL_INTERVAL, Constants.TIMEOUT_FOR_RESOURCE_READINESS,
+            () -> testMethodResources().kafkaConnect().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getStatus()
+                        .getConditions().get(0).getType().equals("Ready"));
+
+        KafkaConnectStatus kafkaStatus = testMethodResources().kafkaConnect().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getStatus();
+
+        assertThat("Kafka connect status should be " + "Ready", kafkaStatus.getConditions().get(0).getType(), is("Ready"));
+
+        testMethodResources().deleteKafkaConnectWithoutWait(kafkaConnect);
+    }
+
+    @Test
+    void testTlsHostnameVerificationWithMirrorMaker() {
+        String sourceKafkaCluster = CLUSTER_NAME + "-source";
+        String targetKafkaCluster = CLUSTER_NAME + "-target";
+
+        testMethodResources().kafkaEphemeral(sourceKafkaCluster, 3, 1).done();
+        testMethodResources().kafkaEphemeral(targetKafkaCluster, 3, 1).done();
```

Comment:
I think one kafka should be enough in that test case. Use same bootstrap as source and target. It will save some time.